app/models.py

```diff
@@ -44,8 +45,10 @@
 SMS_TYPE = 'sms'
 EMAIL_TYPE = 'email'
 LETTER_TYPE = 'letter'
+BROADCAST_TYPE = 'broadcast'
 
-TEMPLATE_TYPES = [SMS_TYPE, EMAIL_TYPE, LETTER_TYPE]
+TEMPLATE_TYPES = [SMS_TYPE, EMAIL_TYPE, LETTER_TYPE, BROADCAST_TYPE]
```

Comment:
Have we made sure the new template_type doesn't affect the front end?

Reply:
The frontend is ready to handle the new type, see: https://github.com/alphagov/notifications-admin/pull/3498